src/content/tutorials/observability/data-exploration/creating-custom-dashboards/_index.md

```diff
@@ -38,7 +42,15 @@ metadata:
   namespace: my-namespace
 ```
 
-### Limitations
+__warning__: the `observability.giantswarm.io/organization` annotation's value must be equal to an existing GrafanaOrganization CR's display name. For more information on Grafana organizations, check the [related page](https://docs.giantswarm.io/tutorials/observability/multi-tenancy/creating-grafana-organization/).
+
+This is the preferred approach as it allows shared dashboards across all installations (if you have multiple ones) and comes with a linting job in the CI pipeline to ensure the dashboard is valid.
+
+### Grafana UI
+
+You can also create your dashboards directly through the Grafana UI. Grafana data is stored in a Postgresql database which is backed up hourly so we can recover the state of Grafana in case of a disaster.
+
+## Limitations
```

Comment:
You have a h2 for limitations here, but an h3 limitations above - is the one above a duplication and you can pull that content down here, or do you maybe want to make the limitations an h4 within the h3 of GitOps and Grafana UI? 

But reading the limitations here, I feel like that belongs to the GitOps approach and not at all to the Grafana UI way, because Grafana automatically creates a UID, right?

Reply:
I removed the `Limitations` subsection and moved its content to the warning part in the GitOps section.